src/ess.jl

```diff
@@ -312,42 +347,95 @@ function ess_rhat(
 end
 
 """
-	copyto_split!(out::AbstractMatrix, x::AbstractMatrix)
+    ess_rhat_bulk(samples::AbstractArray{<:Union{Missing,Real},3}; kwargs...)
+
+Estimate the bulk-effective sample size and bulk-``\\widehat{R}`` values for the `samples` of
+shape `(draws, chains, parameters)`.
 
-Copy the elements of matrix `x` to matrix `out`, in which each column of `x` is split.
+For a description of `kwargs`, see [`ess_rhat`](@ref).
 
-If the number of rows in `x` is odd, the sample at index `(size(x, 1) + 1) / 2` is dropped.
+The bulk-ESS and bulk-``\\widehat{R}`` are variants of ESS and ``\\widehat{R}`` that
+diagnose poor convergence in the bulk of the distribution due to trends or different
+locations of the chains. While it is conceptually related to [`ess_rhat`](@ref) for
+`Statistics.mean`, it is well-defined even if the chains do not have finite variance.
+[^VehtariGelman2021]
```

Comment:
This definitely makes sense for Markdown. For docstrings and comments, I assume the reason for wrapping is so that it's easily readable on GitHub, where wrapping doesn't occur. (Markdown is just rendered as Markdown, so not an issue there). At any rate, BlueStyle wraps docstrings at 92 characters.